Cargo.toml

```diff
@@ -87,3 +87,5 @@ std = [
 ws-client = ["ws"]
 staking-xt = ["std", "staking"]
 
+[patch.crates-io]
```

Comment:
Could you include an explanation why this is necessary? Link to issue is fine ofc

Reply:
Good point, thanks!